napari/components/_tests/test_layers_list.py

```diff
@@ -427,7 +427,10 @@ def test_layers_save_selected(builtins, tmpdir, layer_data_and_types):
 
 
 # the layers fixture is defined in napari/conftest.py
-@pytest.mark.filterwarnings('ignore:`np.int` is a deprecated alias for')
+# TODO: this warning filter can be removed when a new version
+# of napari-svg includes the following PR:
+# https://github.com/napari/napari-svg/pull/38
```

Comment:
napari-svg 0.2.0 is out, but we need to update the test requirements to remove this. Maybe worth leaving until a follow-up PR.